webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -2162,14 +2216,15 @@ class StringFormatter private (
    *                    username.
    * @return the same optional string.
    */
+  @deprecated("Use validateAndEscapeOptionalUsername(Option[String]) instead.")
   def validateAndEscapeOptionalUsername(
     maybeString: Option[String],
     errorFun: => Nothing
   ): Option[String] = // V2 / value objects
-    maybeString match {
-      case Some(s) => Some(validateAndEscapeUsername(s, errorFun))
-      case None    => None
-    }
+    maybeString.map(validateAndEscapeUsername(_).getOrElse(errorFun))
+
+  def validateAndEscapeOptionalUsername(maybeString: Option[String]): Option[String] =
```

Comment:
Consider removing this method and inline the original `validateAndEscapeOptionalUsername` instead.

Reply:
All "optional" methods are gone now 🔥